templates/frontend/parser/articleMainText.tpl

```diff
@@ -176,14 +176,25 @@
 
                 <div class="card article-affiliations" id="article-affiliations">
                     <div class="card-header article-html-views">
-                        <h3 class="card-title">{translate key="plugins.gregg.affiliations"}</h3>
+                        <h3 class="card-title">{translate key="plugins.gregg.authordetails"}</h3>
                     </div>
                     <div class="card-body">
                         {if $article->getAuthors()}
                             {foreach from=$article->getAuthors() item=author key=y}
                                 <p class="card-text">
-                                    <i>{$author->getFullName()|escape}</i><br/>
-                                    {if $author->getLocalizedAffiliation()}{$author->getLocalizedAffiliation()|escape}{else}{translate key="plugins.gregg.no-affiliation"}{/if}
+                                    <i><b>{$author->getFullName()|escape}</b></i><br/>
+									{if $author->getLocalizedBiography()}{$author->getLocalizedBiography()|escape}{/if}
+									{if $author->getLocalizedAffiliation()}{$author->getLocalizedAffiliation()|escape}{else}{translate key="plugins.gregg.no-affiliation"}{/if}
+									<br />
```

Comment:
I usually avoid the `br` tag because it is not semantically clear. Maybe it is better to put every separate element here into span or div?